libraries/AP_PiccoloCAN/AP_PiccoloCAN_Servo.cpp

```diff
@@ -30,6 +30,19 @@ bool AP_PiccoloCAN_Servo::handle_can_frame(AP_HAL::CANFrame &frame)
     if (decodeServo_StatusAPacketStructure(&frame, &status.statusA)) {
         newTelemetry = true;
     } else if (decodeServo_StatusBPacketStructure(&frame, &status.statusB)) {
+        AP_ESC_Telem_Backend::TelemetryData t {
+            .voltage = float(servo.statusB.voltage) * 0.01f,
+            .current = float(servo.statusB.current) * 0.01f,
+            .consumption_mah = 0.0f,
+            .usage_s = 0,
+            .last_update_ms = 0,
+            .temperature_cdeg = int16_t(servo.statusB.temperature * 100),
+        };
+
+        update_telem_data(addr, t,
+                  AP_ESC_Telem_Backend::TelemetryType::CURRENT
+                | AP_ESC_Telem_Backend::TelemetryType::VOLTAGE
+                | AP_ESC_Telem_Backend::TelemetryType::TEMPERATURE);
         newTelemetry = true;
     } else if (decodeServo_FirmwarePacketStructure(&frame, &settings.firmware)) {
     } else if (decodeServo_AddressPacketStructure(&frame, &settings.address)) {
```

Comment:
Looks like this addition is breaking the compile currently.

This change shouldn't be needed as:
 - Servo data is already being logged in AP_PiccoloCAN.cpp
 - Currently other CAN servos (DroneCAN, etc.) aren't reporting voltage/current/temp/etc. over the ESC telem backend for mavlink anyway

Reply:
Thanks, fixed